heft-plugins/heft-jest-plugin/tsconfig.json

```diff
@@ -2,6 +2,6 @@
   "extends": "./node_modules/@rushstack/heft-node-rig/profiles/default/tsconfig-base.json",
 
   "compilerOptions": {
-    "types": ["node"]
+    "types": ["node", "heft-jest"]
```

Comment:
Does this not come from `heft-node-rig`?

Reply:
No it actually doesn't currently. That just specifies `types: []`. This is also true for `@rushstack/heft-web-rig`. I believe this can be updated to put typings at that level if we wanted to do so.

Reply:
Naw, let's leave that as-is for now.

Reply:
Actually, does the rig specify the Jest plugin? It should probably pull the types in then.